firstvoices/backend/urls.py

```diff
@@ -14,6 +15,10 @@
 
 # site-level APIs
 sites_router = NestedSimpleRouter(ROUTER, r"sites", lookup="site")
+sites_router.register(r"characters", CharactersViewSet, basename="characters")
+sites_router.register(
+    r"ignored_characters", IgnoredCharactersViewSet, basename="ignored_characters"
```

Comment:
It might be worth changing this to `ignored-characters` to match other URLs such as `parts-of-speech` for consistency.